### PR TITLE
Bug fixes on Doctor

### PR DESCRIPTION
**Pull request by chenlevy24**

### CODE REVIEW

File: packages/core/src/lib/jdk/JdkHelper.ts [hunk 1]
```diff
@@ -20,6 +20,21 @@ import {Config} from '../Config';
 import * as path from 'path';
 import {executeFile} from '../util';
 
+/**
+ * Returns the Home folder of the jdk.
+ * @param {Config} config The bubblewrap general configuration
+ * @param {NodeJS.Process} process Information from the OS process
+ */
+export function getJavaHome(config: Config, process: NodeJS.Process): string {
```
Comment (andreban) [inline packages/core/src/lib/jdk/JdkHelper.ts:28]: We can declare this as a `static` function in `JdkHelper`.

File: packages/cli/src/lib/cmds/doctor.ts [hunk 2]
```diff
@@ -19,9 +19,10 @@ import {join} from 'path';
 import {existsSync, promises as fsPromises} from 'fs';
 import {loadOrCreateConfig} from '../config';
 import {enUS as messages} from '../strings';
+import {getJavaHome} from '../../../../core/src/lib/jdk/JdkHelper';
```
Comment (andreban) [inline packages/cli/src/lib/cmds/doctor.ts:22]: You are importing classes from `core` by relative path. You need to import using `@bubblewrap/core`. This is causing the build to break.

File: packages/cli/src/lib/cmds/doctor.ts [hunk 3]
```diff
@@ -19,9 +19,10 @@ import {join} from 'path';
...
 
 async function jdkDoctor(config: Config, log: Log): Promise<boolean> {
-  const jdkPath = config.jdkPath;
+  const jdkPath = getJavaHome(config, process);
```
Comment (andreban) [inline packages/cli/src/lib/cmds/doctor.ts:25]: What about moving the entire path validation to a static function in `JdkHelper` and just using it in doctor?

File: packages/cli/src/lib/cmds/doctor.ts (hunk 2 above)
Comment (chenlevy24) [reply packages/cli/src/lib/cmds/doctor.ts:22]: And let the androidSdk check stay in `doctor`? it seems weird a little bit that only one part of the two will be done in `doctor` don't it?

File: packages/cli/src/lib/cmds/doctor.ts (hunk 3 above)
Comment (andreban) [reply packages/cli/src/lib/cmds/doctor.ts:25]: We could move the androidSdk check to AndroidSdkTools, and remove code duplication here: https://github.com/GoogleChromeLabs/bubblewrap/blob/master/packages/core/src/lib/androidSdk/AndroidSdkTools.ts#L44-L46

**Comment by chenlevy24** [comment]
That's weird, in my code I pass both `npm run lint` and `npm run test`

**Pull request closed without merging**